cdap-runtime-ext-dataproc/src/main/resources/gce-dataproc.json

```diff
@@ -63,6 +66,7 @@
           "widget-type": "select",
           "label": "Zone",
           "name": "zone",
+          "description": "Each region has one or more zones. For example, the us-central1 region denotes a region in the Central United States that has zones us-central1-a, us-central1-b, us-central1-c, and us-central1-f",
```

Comment:
How about: "A zone is an isolated location within a region. The fully-qualified name for a zone is made up of <region>-<zone>. For example, the fully-qualified name for zone a in region us-central1 is us-central1-a."